packages/shopify-codemod/transforms/iife-to-ternary-expression.js

```diff
@@ -0,0 +1,36 @@
+export default function iifeToTernaryExpression({source}, {jscodeshift: j}, {printOptions = {}}) {
+  function isIIFE(path) {
+    return j.ArrowFunctionExpression.check(path.node.callee);
+  }
+
+  function calleeContainsIfStatement(path) {
+    return isBlockStatementContainingOnly(path.get('callee', 'body'), j.IfStatement);
+  }
+
+  function ifStatementOnlyReturnsTwoValues(path) {
+    const ifStatement = path.get('callee', 'body', 'body', 0);
+    return isBlockStatementContainingOnly(ifStatement.get('consequent'), j.ReturnStatement) &&
+           isBlockStatementContainingOnly(ifStatement.get('alternate'), j.ReturnStatement);
+  }
+
+  function isBlockStatementContainingOnly(path, childType) {
+    return j.BlockStatement.check(path.node) &&
```

Comment:
Personally not a huge fan of the extra-indents-for-lining-up style, I prefer something like

```js
return (
  foo &&
  bar &&
  baz
);
```

when you need to break it up over multiple lines.